data/mp/stats/weapons.json

```diff
@@ -4450,13 +4450,79 @@
         "numExplosions": 1,
         "recoilValue": 100,
         "rotate": 180,
-        "shortHit": 55,
-        "shortRange": 960,
+        "shortHit": 60,
+        "shortRange": 640,
         "waterGfx": "FXSSplsh.PIE",
         "weaponClass": "KINETIC",
         "weaponEffect": "ANTI PERSONNEL",
         "weaponSubClass": "MACHINE GUN",
         "weaponWav": "mgbar3.ogg",
         "weight": 1
+    },
+    "TUTMG": {
+        "shortRange": 512,
```

Comment:
Do not include these incomplete entries for obsolete weapons.

Reply:
Oh ok. I can remove this.